samples/sleep/sleep.yaml

```diff
@@ -37,6 +37,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: sleep
+  labels:
```

Comment:
Sleep doesn't need this, because there are no version and the label name is the same as the deployment name.

Reply:
Done.